constants/index.js

```diff
@@ -56,7 +58,18 @@ const BAD_REQUEST_CODE = 400,
   booleanError = fieldName => `${fieldName} must either be true or false`,
   GET_USER_RATING = 'Get user rating successfully',
   GET_USER_RATING_ERROR = 'One or both of articleId and userId not found',
-  RATING_CREATED = 'Your rating have been saved succesfully';
+  RATING_CREATED = 'Your rating have been saved succesfully',
+  ALREADY_FOLLOWING_AUTHOR = 'You are already following this author',
+  FOLLOW_SELF = 'You cannot follow yourself',
+  NOW_FOLLOWING_USER = 'You are now following this author',
+  UNFOLLOW_USER_MESSAGE = 'You have successfully unfollowed this Author',
+  UNFOLLOW_SELF = 'You cannot unfollow yourself',
+  NOT_FOUND_CODE_MESSAGE = 'User not found',
+  NO_FOLLOWING = 'You currently don\'t have a following',
```

Comment:
Kindly make the response messages more descriptive

Reply:
Done